metadata-ingestion/src/datahub/ingestion/source/unity/hive_metastore_proxy.py

```diff
@@ -100,14 +100,47 @@ def hive_metastore_schemas(self, catalog: Catalog) -> Iterable[Schema]:
             )
 
     def hive_metastore_tables(self, schema: Schema) -> Iterable[Table]:
-        views = self.inspector.get_view_names(schema.name)
+        views = self.get_view_names(schema.name)
```

Comment:
This is in fact fixed in databricks-sql-connector in versions >=3.0.0 . However, since databricks-sql-connector is pinned to <3.0.0 due to requirement of SQLAlchemy > 2.0.21. cc: @hsheth2

Reply:
can you add a TODO or NOTE saying this?

it might be a bit tricky for us to refactor our CI such that we can start using sqlalchemy 2.x